src/core/abstracts/command/command.abstract.ts

```diff
@@ -14,12 +14,11 @@ export class BaseCommand<K extends CmdType> extends Base {
     throw new Error("Cannot be invoked in parent class");
   }
 
-  sendError(
-    error: string,
-    arg: CmdArg<CmdType.SLASH_COMMAND>,
-    isExpected = true
-  ) {
+  sendError(error: string, arg: CmdArg<K>, isExpected = true) {
     try {
+      const isSlash = arg.type === InteractionType.ApplicationCommand;
```

Comment:
@shysecre Is we okay with check like this? Or you have more ✨fancy✨ solution for this?